tests/stat/test_stat.c

```diff
@@ -161,6 +164,17 @@ void test() {
   assert(s.st_blocks == 1);
 #endif
 
+  // create and unlink files inside a directory and check that mtime updates
+  mkdir("folder/subdir", 0777);
+  utime("folder/subdir", &t);
+  create_file("folder/subdir/file", "abcdef", 0777);
+  err = stat("folder/subdir", &s);
+  assert(s.st_mtime != 1200000000);
+  utime("folder/subdir", &t);
+  unlink("folder/subdir/file");
+  err = stat("folder/subdir", &s);
+  assert(s.st_mtime != 1200000000);
```

Comment:
Should we assert that its near to the current time?   Maybe that is too timing sensitive?   How about at least replacing != with strictly greater than?

lgtm either way... let me know if you'd prefer to land this as is.

Reply:
Being different from 2008-01-10T22:20:00 should be enough.